docs/sphinx/source/whatsnew/v0.9.2.rst

```diff
@@ -79,3 +76,6 @@ Contributors
 * Jules Chéron (:ghuser:`jules-ch`)
 * Kurt Rhee (:ghuser:`kurt-rhee`)
 * Will Hobbs (:ghuser:`williamhobbs`)
+* Stephen Schneider (:ghuser:`sjschneider`)
+* :ghuser:`Kaesekopf`
+* :ghuser:`hf-kklein`
```

Comment:
We try to acknowledge people that have contributed to discussions in issues/PRs. Does this list include a quick read for those people? If not I can help.

Reply:
@campanelli-sunpower
@adriesse 
@KWagnerCPR
@Kaesekopf
@mikofski

Reply:
Thanks!  Sloppy of me.

Reply:
It would be nice if GitHub could make that automatic. As it is, I think the best thing we can do is to [add more co-authors to commits](https://docs.github.com/en/pull-requests/committing-changes-to-your-project/creating-and-editing-commits/creating-a-commit-with-multiple-authors#creating-co-authored-commits-on-github). See [Hynek's TIL](https://hynek.me/til/easier-crediting-contributors-github/) for how to do that easily. Then use a tool for automating the contributor list.